index.d.ts

```diff
@@ -181,6 +181,7 @@ declare module 'react-native-twilio-video-webrtc' {
     enableVideo?: boolean;
     enableRemoteAudio?: boolean;
     enableH264Codec?: boolean;
+    enableAutomaticSubscription?: boolean;
```

Comment:
Let's provide the default value from js explicitly in `TwilioVideo.connect` implementation